python/turbine_models/custom_models/stateless_llama.py

```diff
@@ -130,7 +142,7 @@ class StateUpdateModule(CompiledModule):
         else:
             params = export_parameters(mod)
         global_state = export_global(
-            abstractify(global_pkv), uninitialized=True, mutable=True
+            abstractify(global_pkv), uninitialized=False, mutable=True
```

Comment:
Is this change necessary for this patch? We were initializing this value later and adds a bunch of unnecessary tensor info to the mlir

Reply:
ah good catch, was debugging something with this one.